### DISCUSSION TITLE
最新dev里分支中的包与使用手册pdf中所注不同，如缺少tabularx

### DISCUSSION THREAD

**Discussion by superjdg** [obster-y/XJTU-thesis#112] [Q&A]
直接替换原有的cls文件后表格部分编译显示缺少包{tabularx}，导致编译失败

**Comment by obster-y** [comment]
之前准备写个更新指南，因为更新后把 tabular 换成了 tabularray，也就是说所有的表格都得换一下写法，建议看一下：

https://github.com/obster-y/XJTU-thesis/blob/74e924d2c5541598a5109424531a92b385954a05/Main_Spine/c2.tex#L173-L178

换得内容不多，换起来也很快

**Reply by obster-y** [reply]
这两天正在赶新的 wiki 和使用说明，所以刚好这个时间点没赶上

**Reply by obster-y** [reply]
真的是有丶巧，因为我刚把 Wiki 写完，已经推上去了，请注意查看

**Reply by superjdg** [reply]
感谢解答！
